rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2587,6 +2587,56 @@ void voteFederationChange_addSameFederatorPublicKeyBySameAuthorizerTwice_returns
 
         }
 
+        @Test
+        void voteFederationChange_addMultiFederatorPublicKeyWithDifferentMstKey_returnsSuccessResponseCodeAndFedSizeZero() {
+
+            // Arrange
+
+            BtcECKey expectedBtcECKey1 = new BtcECKey();
+            ECKey expectedRskKey = new ECKey();
+            ECKey expectedMstKey = new ECKey();
```

Comment:
why call them _expected_? you are not checking for those values anywhere, just using them

Reply:
Updated.